backup.go

```diff
@@ -104,11 +104,17 @@ func (b *BackupEngine) UnsafeGetBackupEngine() unsafe.Pointer {
 	return unsafe.Pointer(b.c)
 }
 
-// CreateNewBackup takes a new backup from db.
-func (b *BackupEngine) CreateNewBackup(db *DB) error {
+// CreateNewBackupFlush takes a new backup from db. If flush is set to true,
+// it flushes the WAL before taking the backup.
+func (b *BackupEngine) CreateNewBackupFlush(db *DB, flush bool) error {
 	var cErr *C.char
 
-	C.rocksdb_backup_engine_create_new_backup(b.c, db.c, &cErr)
+	var cFlush = C.uchar(0)
```

Comment:
You can remove this temp var and just use `boolToChar(flush)` in the call:
`C.rocksdb_backup_engine_create_new_backup_flush(b.c, db.c, boolToChar(flush), &cErr)`

Reply:
Done!